src/controllers/users-controller.ts

```diff
@@ -45,4 +47,31 @@ export default class UsersController extends AbstractController {
     await this.users.remove(user);
     return { removed: true, id };
   }
+
+  @PUT
+  @Path('/:id')
+  @Response<{ id: string; removed: boolean }>(200, 'Password succesfully updated.')
```

Comment:
I'd merged a master branch in here for now. And with more I updated the methods with the `typescript-rest` used, you'll need to get it right.